### PR TITLE
Matt

### PR DESCRIPTION
**Pull request by matt-voror**

### CODE REVIEW

File: im_library/src/helpers/StringManipulators.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export function toTitleCase(string: string) {
```
Comment (kkardatos) [inline im_library/src/helpers/StringManipulators.ts:1]: double export not needed

File: ui/src/components/directory/landingPage/PieChartDashCard.vue [hunk 2]
```diff
@@ -44,7 +46,7 @@ function setChartSize(): void {
   const reportTable = document.getElementsByClassName("report-table")[0] as HTMLElement;
   const container = document.getElementById(props.id) as HTMLElement;
   if (!container) {
-    LoggerService.error(undefined, `Failed to set chart size for element id: ${props.id}`);
+    log.error(() => `Failed to set chart size for element id: ${props.id}`);
```
Comment (kkardatos) [inline ui/src/components/directory/landingPage/PieChartDashCard.vue:49]: does that need to be within a () => 'a' instead of just a string?

File: ui/src/components/directory/landingPage/PieChartDashCard.vue [hunk 3]
```diff
@@ -44,7 +46,7 @@ function setChartSize(): void {
...
     return;
```
Comment (kkardatos) [inline ui/src/components/directory/landingPage/PieChartDashCard.vue:50]: (not part of this pr changes) but this is bad practice - should refactor to if/else instead to avoid executing code that is not relevant or explicitly returning false/undefined if the return value is used

File: ui/src/components/directory/landingPage/PieChartDashCard.vue (hunk 2 above)
Comment (matt-voror) [reply ui/src/components/directory/landingPage/PieChartDashCard.vue:49]: Examples on the repo show it as arrow function. Not sure if necessary though as might just be building the string.
